lib/model/query/analytics.js

```diff
@@ -420,103 +420,116 @@ group by f."projectId"`);
+       , COALESCE(creates.create_api_recent       , 0) AS num_entity_create_api_recent
+       , COALESCE(bulk_creates.total              , 0) AS num_entity_create_bulk_total
+       , COALESCE(bulk_creates.recent             , 0) AS num_entity_create_bulk_recent
+  FROM datasets ds
+    LEFT JOIN (
+      SELECT "datasetId"
+           , COUNT(id) AS num_entities_total
+           , COUNT(CASE WHEN "createdAt" >= current_date - 45 THEN 1 END) as num_entities_recent
+           , COUNT(CASE WHEN "updatedAt" IS NOT NULL          THEN 1 END) AS num_entities_updated_total
+           , COUNT(CASE WHEN "updatedAt" >= current_date - 45 THEN 1 END) AS num_entities_updated_recent
+        FROM entities
+        GROUP BY "datasetId"
+    ) AS entz ON entz."datasetId" = ds.id
+    LEFT JOIN (
+      SELECT dfd."datasetId"
+           , COUNT(fd."formId") AS num_creation_forms
+        FROM dataset_form_defs AS dfd
+          JOIN form_defs AS fd ON fd.id = dfd."formDefId"
+        GROUP BY dfd."datasetId"
+    ) AS fd ON fd."datasetId" = ds.id
+    LEFT JOIN (
+      SELECT fa."datasetId"
+           , COUNT(fa."formId") AS num_followup_forms
+        FROM form_attachments AS fa
+          JOIN forms AS f ON f.id = fa."formId"
+        WHERE f."currentDefId" IS NOT NULL
+        GROUP BY fa."datasetId"
+    ) AS fa ON fa."datasetId" = ds.id
+    LEFT JOIN (
+      SELECT
+        COUNT (a.details -> 'submissionId'::TEXT) total,
+        SUM (CASE WHEN a."loggedAt" >= current_date - 45 THEN 1 ELSE 0 END) recent,
+        dfd."datasetId"
+      FROM audits a
+        JOIN submissions s ON CAST((a.details ->> 'submissionId'::TEXT) AS integer) = s.id
+        JOIN submission_defs sd ON sd."submissionId" = s.id AND sd."current"
+        JOIN dataset_form_defs dfd ON sd."formDefId" = dfd."formDefId"
+      WHERE a."action" = 'entity.error'
+      GROUP BY dfd."datasetId"
+    ) AS errors ON ds.id = errors."datasetId"
+    LEFT JOIN (
+      SELECT
+        COUNT (*) total,
+        SUM (CASE WHEN a."loggedAt" >= current_date - 45 THEN 1 ELSE 0 END) recent,
+        SUM (CASE WHEN a."details"->'submissionDefId' IS NOT NULL THEN 1 ELSE 0 END) update_sub_total,
+        SUM (CASE WHEN a."details"->'submissionDefId' IS NOT NULL AND a."loggedAt" >= current_date - 45 THEN 1 ELSE 0 END) update_sub_recent,
+        SUM (CASE WHEN a."details"->'submissionDefId' IS NULL THEN 1 ELSE 0 END) update_api_total,
+        SUM (CASE WHEN a."details"->'submissionDefId' IS NULL AND a."loggedAt" >= current_date - 45 THEN 1 ELSE 0 END) update_api_recent,
+        e."datasetId"
+      FROM audits a
+        JOIN entities e on CAST((a.details ->> 'entityId'::TEXT) AS integer) = e.id
+      WHERE a."action" = 'entity.update.version'
+      GROUP BY e."datasetId"
+    ) as updates ON ds.id = updates."datasetId"
+    LEFT JOIN (
+      SELECT
+        -- not used: COUNT (*) total,
+        -- not used: SUM (CASE WHEN a."loggedAt" >= current_date - 45 THEN 1 ELSE 0 END) recent,
```

Comment:
@ktuite any thoughts on these?  They're currently computed but not used in the final `SELECT`.  Perhaps they should be used rather than removed?

Reply:
They can be removed. I can't remember why I added this (maybe I was planning to use it?) but there isn't a usage metric for this because we started counting a finer grain metric (creation counts split by source type).